tests/acl/test_stress_acl.py

```diff
@@ -24,18 +25,45 @@
 # Template json file used to test scale rules
 STRESS_ACL_TABLE_TEMPLATE = "acl/templates/acltb_test_stress_acl_table.j2"
 STRESS_ACL_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_rules.j2"
-STRESS_ACL_READD_RULE_TEMPLATE = "acl/templates/acltb_test_stress_acl_readd_rules.j2"
-DEL_STRESS_ACL_TABLE_TEMPLATE = "acl/templates/del_acltb_test_stress_acl_table.j2"
 STRESS_ACL_TABLE_JSON_FILE = "/tmp/acltb_test_stress_acl_table.json"
 STRESS_ACL_RULE_JSON_FILE = "/tmp/acltb_test_stress_acl_rules.json"
+DEL_STRESS_ACL_TABLE_TEMPLATE = "acl/templates/del_acltb_test_stress_acl_table.j2"
 DEL_STRESS_ACL_TABLE_JSON_FILE = "/tmp/del_acltb_test_stress_acl_table.json"
 
+STRESS_ACL_50_RULES_JSON_SRC = "acl/templates/acltb_test_stress_50_plus_rules.json"
+STRESS_ACL_50_RULES_JSON_DST = "/tmp/acltb_test_stress_50_plus_rules.json"
+
 LOG_EXPECT_ACL_TABLE_CREATE_RE = ".*Created ACL table.*"
 LOG_EXPECT_ACL_RULE_FAILED_RE = ".*Failed to create ACL rule.*"
 
 ACL_RULE_NUMS = 10
 
 
+@pytest.fixture(scope="module")
+def setup_table_and_rules(rand_selected_dut, prepare_test_port):
+
+    logger.debug('Setting up rules')
+    _, _, dut_port = prepare_test_port
+    logger.debug(f'dut_port: {dut_port}')
+    table_name = 'STRESS_ACL_50'
+    # Add table
+    cmd_add_table = f"config acl add table {table_name} L3 -s ingress -p {dut_port}"
+    rand_selected_dut.shell(cmd_add_table)
+    logger.debug('Table created')
+    # Copy rules file and add rules
+    rand_selected_dut.copy(src=STRESS_ACL_50_RULES_JSON_SRC, dest=STRESS_ACL_50_RULES_JSON_DST, mode="0755")
+    cmd_add_rules = f"sonic-cfggen -j {STRESS_ACL_50_RULES_JSON_DST} -w"
+    rand_selected_dut.shell(cmd_add_rules)
```

Comment:
Please check the status of ACL rules to confirm the rules are programmed to ASIC. This can be done by checking the `status` in `show acl rule STRESS_ACL_50`

Reply:
I have added a change to verify the ACL rules have been applied correctly.